plugins/node/opentelemetry-instrumentation-mongodb/test/mongodb.test.ts

```diff
@@ -244,6 +249,93 @@ describe('MongoDBInstrumentation', () => {
     });
   });
 
+  describe('when specifying a responseHook configuration', () => {
+    const dataAttributeName = 'mongodb_data';
+    beforeEach(() => {
+      memoryExporter.reset();
+    });
+
+    describe('with a valid function', () => {
+      beforeEach(() => {
+        create({
+          responseHook: (span: Span, result: CommandResult) => {
+            span.setAttribute(dataAttributeName, JSON.stringify(result.result));
+          },
+        });
+      });
+
+      it('should attach response hook data to the resulting span for insert function', done => {
+        const insertData = [{ a: 1 }, { a: 2 }, { a: 3 }];
+        const span = provider.getTracer('default').startSpan('insertRootSpan');
+        context.with(trace.setSpan(context.active(), span), () => {
+          collection.insertMany(insertData, (err, result) => {
+            span.end();
+            assert.ifError(err);
+            const spans = memoryExporter.getFinishedSpans();
+            const insertSpan = spans[0];
+
+            assert.strictEqual(
+              insertSpan.attributes[dataAttributeName],
+              JSON.stringify(result.result, Object.keys(result.result).sort())
+            );
+
+            memoryExporter.reset();
```

Comment:
why repeat it in every test? looks like it's already handled in the `beforeEach` call

Reply:
Correct, tests are passing without this call as well.
Removed.
Thanks!